### PR TITLE
fix metrics defer issue

### PR DESCRIPTION
**Pull request by carmark**
refer: https://play.golang.org/p/8w-CbiZILWH

### CODE REVIEW

**Comment by k82cn** [comment]
btw, please signed the PR :)

**Comment by carmark** [comment]
@k82cn done.

File: pkg/scheduler/scheduler.go [hunk 1]
```diff
@@ -70,7 +70,6 @@ func (pc *Scheduler) runOnce() {
 	klog.V(4).Infof("Start scheduling ...")
 	scheduleStartTime := time.Now()
 	defer klog.V(4).Infof("End scheduling ...")
-	defer metrics.UpdateE2eDuration(metrics.Duration(scheduleStartTime))
```
Comment (hzxuzhonghu) [inline pkg/scheduler/scheduler.go:73]: good catch

**Pull request merged**